### DISCUSSION TITLE
Is there any way to remove part of the blank area on the website foot?

### DISCUSSION THREAD

**Discussion by aluopy** [mmistakes/minimal-mistakes#3687] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/69625113/173730406-4fddf103-e075-426a-9fc8-3d1d20b1895a.png)

Is there any way to remove part of the blank area on the website foot? 
This seems to be an advertisement by a third -party advertiser。

**Comment by iBug** [comment, marked as answer]
There's no built-in "blank area" unless you copied something strange from an unknown source.

Unless you provide a website and repository for us to look at, there's nothing we can diagnose further.

**Reply by aluopy** [reply]
Has been solved, thank you
